terraform/eks/container-insights-agent/config_map_fargate.yml

```diff
@@ -0,0 +1,442 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: adot-collector-config
+  namespace: default
+  labels:
+    app: aws-adot
+    component: adot-collector-config
+data:
+  adot-collector-config: |
+    receivers:
```

Comment:
Is this the same config as in `aoc_fargate_deploy.yml`?

Reply:
Yes the first file contains the fill config. I can delete it. Sorry I was using it as a reference.